src/explore-education-statistics-common/src/components/Tabs.tsx

```diff
@@ -14,9 +14,10 @@ import TabsSection, { TabsSectionProps } from './TabsSection';
 interface Props {
   children: ReactNode;
   id: string;
+  onToggle?: (statisticsTab: { id: string; title: string }) => void;
```

Comment:
`statisticsTab` doesn't sound right given that Tabs is a generic component. It should be something like `section`